### ISSUE TITLE
child-row toggler options

### ISSUE THREAD

**Issue by stevenschemers** [matfish2/vue-tables-2#679]
Firstly, I'd like to say thank you for creating this data table. It's by far the best option for my projects with a ton of flexibility. 

That being said, It would be wonderful if the parent cells containing the child row toggle `span` were given a class to identify the cells. Or even a `showChildRowToggler: false` option.

Currently, the best way to hide the toggler cells is to select the first cell in the table rows but this does not always work well when taking a more dynamic approach to creating/extending table configs. 

I use this table on several pages, some of which have child rows and some that don't. To avoid duplicating the css on the multiple pages which need the toggler to be hidden a simple parent class on the cell would make this easier.

Current css for hiding child row toggler
```
.VueTables__table tr > th:first-child, 
.VueTables__table tr > td:first-child {
    display: none;
}  
```

Preferred css for hiding child row  toggler
```
.VueTables__table tr > th.VueTables__child-row-toggler-cell, 
.VueTables__table tr > td:VueTables__child-row-toggler-cell {
    display: none;
}  
```

Relevant change for visiblility option
```
if (this.hasChildRow && this.showChildRowToggler) {
        var childRowToggler = <td><span on-click={this.toggleChildRow.bind(this,row[rowKey])} class={`VueTables__child-row-toggler ` + this.childRowTogglerClass(row[rowKey])}></span></td>;
        if (this.opts.childRowTogglerFirst) columns.push(childRowToggler);
      }
```

**Comment by stevenschemers** [comment]
Here's my PR which added an option for the toggler rather than a parent class. https://github.com/matfish2/vue-tables-2/pull/680

**Comment by matfish2** [comment]
closed by PR #680. hat tip to @stevenschemers!
